### DISCUSSION TITLE
Comment introduire une perioe d'arrêt maladie ?

### DISCUSSION THREAD

**Discussion by zaki076** [betagouv/mon-entreprise#2923] [Q&A] [answered]
Bonjours, j'utilise le simulateur "salarié" et je trouve pas comment introduire une période d'arrêt maladie. tandis que le simulateur prévoit les informations concernant le "IJ" [L'indemnité journalière percu pendant votre arrêt de travail](https://mon-entreprise.urssaf.fr/documentation/protection-sociale/maladie/arr%C3%AAt-maladie/salari%C3%A9/indemnit%C3%A9s)
j'espère trouver de l'aide, merci d'avance

**Comment by liliced** [comment, marked as answer]
Bonjour,

Non il n'est pas possible d'inclure de période d'arrêt maladie dans le simulateur pour salarié⋅es. Les informations sur les IJ sont là pour expliquer les droits obtenus par le ou la salariée.
